test/e2e/es_cluster_logging.go

```diff
@@ -240,6 +240,18 @@ func ClusterLevelLoggingWithElasticsearch(f *Framework) {
 	missing := 0
 	expected := nodeCount * countTo
 	for start := time.Now(); time.Since(start) < graceTime; time.Sleep(10 * time.Second) {
+
+		// Debugging code to report the status of the elasticsearch logging endpoints.
+		esPods, err := f.Client.Pods(api.NamespaceDefault).List(labels.Set{"k8s-app": "elasticsearch-logging"}.AsSelector(), fields.Everything())
+		if err != nil {
+			Logf("Failed to find any elasticsearch logging nodes: %v", err)
```

Comment:
I think this is a failure. If no pods were without any errors occurring then err will be `nil` and `esPods.Items` will be an empty slice. However, if some unexpected error occurs trying to get the pod information then something unexpected has gone wrong and `err` may tell us why.